src/components/speaker.tsx

```diff
@@ -0,0 +1,60 @@
+import type { Speaker } from "@/content/speakers/speaker"
+import { ICONS } from "./icons"
+
+export const Speakers = (props: Readonly<{ speaker: Speaker }>) => {
+  return (
+    <div className="flex flex-col gap-2">
+      <div
+        className="mx-auto aspect-square rounded-lg overflow-hidden"
+      >
+        <img
+          src={props.speaker.imageUrl}
+          alt={`${props.speaker.name}-profile-picture`}
```

Comment:
Alt should be readable for accessibility, providing `-profile-picture` as a suffix with `-` will make it unreadable.

```suggestion
          alt={`${props.speaker.name} profile picture`}
```